app/assets/stylesheets/pages/_home.scss

```diff
@@ -0,0 +1,47 @@
+#hero {
+  $medium-gray: #999 !default;
+  $dark-purple: #0E0866 !default;
+  $base-border-radius: 3px !default;
+  $action-color: #477DCA !default;
+  $large-screen: em(860) !default;
+  $hero-color: $medium-gray;
+
+  background: white no-repeat top;
+  background-size: cover;
+  padding: 2em 0;
+
+  .hero-inner {
+    @include outer-container();
+    @include clearfix;
+    margin: auto;
+    padding: 3.5em;
+    text-align: center;
+
+    .hero-copy {
+      text-align: center;
+
+      h1 {
+        color: $dark-purple;
+        font-size: 1.6em;
+        font-weight: 300;
+        margin-bottom: 0.5em;
+
+        @include media($large-screen) {
+          font-size: 1.8em;
+        }
+      }
+
+      p {
+        color: $hero-color;
+        line-height: 1.4em;
+        font-weight: 300;
+        margin: 0 auto 2em auto;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 auto 2em` instead of `0 auto 2em auto`